src/Symfony/Component/Messenger/Retry/MultiplierRetryStrategy.php

```diff
@@ -63,15 +63,23 @@ public function __construct(int $maxRetries = 3, int $delayMilliseconds = 1000,
         $this->maxDelayMilliseconds = $maxDelayMilliseconds;
     }
 
-    public function isRetryable(Envelope $message): bool
+    public function isRetryable(Envelope $message/*, \Throwable $throwable*/): bool
```

Comment:
the new argument should default to null, this is required to not break BC hardly in v6
also, all implementation must add the `@param` annotation, even if it duplicates the parent.
This tells "I did add the argument, don't throw a deprecation about this"

Reply:
Does that mean that with the `@param` in the implementation the `trigger_deprecation` I added is now useless ?

Reply:
actually, since the class is final, we can the argument for real on this one!